moped-editor/src/views/projects/projectView/ProjectComponentEdit.js

```diff
@@ -895,7 +893,7 @@ const ProjectComponentEdit = ({
                         rows={4}
                         defaultValue=""
                         variant="filled"
-                        value={componentDescription}
+                        value={componentDescription ?? ""}
```

Comment:
did this to clear an error that was showing up in the console, not having values in fields be null